app/src/main/java/app/model/servicedefinition/ServiceDefinition.java

```diff
@@ -0,0 +1,75 @@
+// Copyright 2023 Libre311 Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app.model.servicedefinition;
+
+
+import app.model.service.Service;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "service_definitions")
+public class ServiceDefinition {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(optional = false)
```

Comment:
In the Service we have: 

```
class Service {
    @OneToOne
    private ServiceDefinition serviceDefinition;
}
```

But here we have 
```
ServiceDefinition {
    @ManyToOne(optional = false)
    private Service service;
}
```

Shouldn't it be either ` @OneToMany and @ManyToOne ` or `@OneToOne` across both?

Reply:
That's right. I'd say `@OneToOne` across would make more sense since each service can have a unique service definition.